src/network/networkd-address.h

```diff
@@ -53,6 +53,11 @@ struct Address {
         union in_addr_union in_addr_peer;
 
         bool ip_masquerade_done:1;
+        bool home;
+        bool nodad;
+        bool mngtmpaddr;
+        bool noprefixroute;
+        bool autojoin;
```

Comment:
Those should be defined as `bool xxx:1`, similarly to `ip_masquerade_done`.

dad→duplicate_address_detection
mngtmpaddress→manage_temporary_address
noprefixroute→prefix_route